js/templates/message-list-item.html

```diff
@@ -5,7 +5,7 @@
 	<div class="mail-message-header">
 		<div class="sender-image avatardiv">
 			{{#if senderImage}}
-			<img src="{{senderImage}}" width="32px" height="32px" />
+			<img class="avatar" src="{{senderImage}}" width="32px" height="32px" />
```

Comment:
It would probably be better to not use .avatar here, since that will cause the avatar jquery plugin to jump in on that img-tag as well.